client/blocks/upwork-stats-nudge/style.scss

```diff
@@ -0,0 +1,78 @@
+.card.upwork-stats-nudge {
+	padding: 0; // allow section header to perfectly sit over the top of the card
+}
+
+.upwork-stats-nudge__body {
+	align-items: flex-end;
+	display: flex;
+	flex-direction: row;
```

Comment:
We could remove that line since `row` is the default value for `flex-direction`.

Reply:
Pefect